omero/developers/Python.rst

```diff
@@ -5,7 +5,9 @@ To access the OMERO.server Python API, you need to install the Python client
 libraries.
 
 From OMERO 5.6.0 release, the client library ``omero-py`` supports Python 3 and
-can be installed via ``pip``. We recommend you use a virtual environment:
+is available on PyPI_ and Conda_. We recommend you use a virtual environment to install the client library:
```

Comment:
The virtual environment is only if not conda, correct?

Reply:
I took it as if conda is also creating a venv, only conda-style-venv - is it a wrong take on the matters ? If yes, then reformulation is indeed needed.